src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java

```diff
@@ -14,6 +14,7 @@
 import org.springframework.web.util.UriComponentsBuilder;
 import uk.gov.hmcts.reform.coh.controller.decision.*;
 import uk.gov.hmcts.reform.coh.controller.decisionreplies.*;
+import uk.gov.hmcts.reform.coh.controller.utils.CohUriBuilder;
 import uk.gov.hmcts.reform.coh.controller.validators.DecisionRequestValidator;
 import uk.gov.hmcts.reform.coh.controller.validators.Validation;
 import uk.gov.hmcts.reform.coh.controller.validators.ValidationResult;
```

Comment:
Maybe you should also refactor the GET Decision Reply URI on line 252?

Reply:
Done